ui/raidboss/data/triggers/ridorana_lighthouse.js

```diff
@@ -540,7 +540,8 @@
         'White Breath': 'Kalkatem',
 
         // FIXME:
-        'Fast Hands': 'Fast Hands',
+        'Fast Hands': 'Schnelle Hände',
```

Comment:
Somehow during your merge, it looks like you lost a bunch of the translations that you did in this file.